### PR TITLE
Adding FEWL token to Mumbai whitelist

### PR DESCRIPTION
**Pull request by KevinKons**

### CODE REVIEW

File: packages/indexer/src/config/network.ts [hunk 1]
```diff
@@ -945,6 +945,17 @@ export const getNetworkSettings = (): NetworkSettings => {
             },
           },
         },
+        whitelistedCurrencies: new Map([
+          [
+            "0x9E9Fce924Fe52869d13944E9eef02e4DB0B2DB7d",
```
Comment (nofir) [inline packages/indexer/src/config/network.ts:950]: - the address needs to be in lowercase (in both places)/
- you can also specify a image for the currency, is that not available?

**Review by nofir — CHANGES_REQUESTED** [review]

**Review by nofir — APPROVED** [review]

**Comment by KevinKons** [comment]
Hi @nofir!
How long does it takes for the changes to take effect? I'm still facing the same error that I was facing before
```
Error: Request failed with status code 400: failed-to-convert-price
```

**Comment by nofir** [comment]
@KevinKons can you try again? are you still getting the error?

**Comment by KevinKons** [comment]
> @KevinKons can you try again? are you still getting the error? can you share the request payload?
@nofir 

This is the payload:
```
{
   "order":{
      "kind":"seaport-v1.5",
      "data":{
         "kind":"single-token",
         "offerer":"0xcdebae039a733111a11f3786e4f2dd8856c940a1",
         "zone":"0x0000000000000000000000000000000000000000",
         "offer":[
            {
               "itemType":3,
               "token":"0xd6557ad6de7b7f0a6c465549cbd077f739205fd8",
               "identifierOrCriteria":"1",
               "startAmount":"1",
               "endAmount":"1"
            }
         ],
         "consideration":[
            {
               "itemType":1,
               "token":"0x9e9fce924fe52869d13944e9eef02e4db0b2db7d",
               "identifierOrCriteria":"0",
               "startAmount":"1000000000000000000",
               "endAmount":"1000000000000000000",
               "recipient":"0xcdebae039a733111a11f3786e4f2dd8856c940a1"
            }
         ],
         "orderType":1,
         "startTime":1696449257,
         "endTime":3913861900,
         "zoneHash":"0x0000000000000000000000000000000000000000000000000000000000000000",
         "salt":"0x6c933bed1d4da48b00000000000000008b2d3691d0cadeca1522628b7805dffa",
         "conduitKey":"0x0000007b02230091a7ed01230072f7006a004d60a8d4e71d599b8104250f0000",
         "counter":"0",
         "signature":"0x0000000000000000000000000000000000000000000000000000000000000000"
      }
   },
   "orderbook":"reservoir",
   "source":"metaflyers.xyz"
}
```

This is the response:
```
{
    "statusCode": 400,
    "error": "Bad Request",
    "message": "failed-to-convert-price",
    "orderId": "0x951b801aa992f122e01b8cb80fdf3fb729b9afdbbd5f90b284a7554c432d73df"
}
```

**Comment by nofir** [comment]
@KevinKons hi, sorry for the delay, we need to debug it, not sure yet as its usually a pretty standard flow. i'll keep you updated.

**Comment by KevinKons** [comment]
> @KevinKons hi, sorry for the delay, we need to debug it, not sure yet as its usually a pretty standard flow. i'll keep you updated.

Thank you!

**Comment by nofir** [comment]
@KevinKons We found the issue and deployed a fix. You should be able to test again.

**Comment by KevinKons** [comment]
Thanks @nofir !

**Pull request merged**